nservicebus/scalability-and-ha/scale-out.md

```diff
@@ -0,0 +1,74 @@
+
+Up to version 5 (inclusive) both SQL Server and RabbitMQ transports scale out by adding more receivers to the same queue, taking advantage of the competing consumers capability built into these transports. All the instances feeding off the queue have same *endpoint name* and *address* so they appear to the outside world as a single instance.
+
+The benefit of this approach is zero configuration. New instances can be added by `xcopy`-ing the deployment folder. The potential downside is that total throughput of the endpoint is capped to the maximum throughput of a single queue in the underlying infrastructure. 
+
+
+### Azure Storage Queues and Azure Service Bus
+
+Up to version 5 (inclusive) both Azure transports behave similarly to other broker transports (SQL Server and RabbitMQ): they scale out by adding receivers to a single queue. An additional feature of Azure transports is ability to use individualized queues (based on the *role instance ID*)
+
+snippet: UniqueQueuePerEndpointInstance
+
+This feature is, however, *not intended* for scaling out as there is no built-in distribution mechanism between that would route messages to the individualized queues.
+
+## Version 6 and above
+
+Version 6 of NServiceBus comes with a unified scalability model which is based on the concept of endpoint instance ID. Each deployment of NServiceBus can (but does not have to) be assigned an *instance ID*. Instance IDs can be shared between several deployments. 
+
+When instance ID is assigned, NServiceBus spins up an additional receiver for the queue which name is based on that ID, e.g. if the endpoint name is `Sales` and instance ID is `Green` that instance will try to receive from queues `Sales` and `Sales-Green` (actual queue names depend on the underlying transport).
+
+It is up to the sender to choose if it is going to treat the endpoint as a whole (and send its messages to `Sales` queue) or address individual instances (e.g. `Sales-Red`, `Sales-Green`, `Sales-Blue`). In the latter case the sender will use round-robin algorithm for choosing destinations of outgoing messages.
+
+
+### MSMQ
+
+When using MSMQ different instances are usually deployed to different (virtual) machines. The following routing file (see [file-based routing](/nservicebus/messaging/file-based-routing.md)) shows scaling out of the `Sales` endpoint.
+
+snippet:Routing-FileBased-MSMQ
+
+The corresponding logical routing is
+
+snippet:Routing-StaticRoutes-Endpoint
+
+NOTE: System administrators are able to spin-up new instances of the endpoint should the load increase and the only requirement is adding an entry to the routing file. No changes in the source code are required.
+
+
+### Broker transports
+
+The main difference when using broker transports is that queues are not attached to machines but are rather maintained by a central server (or cluster of servers). When running on a broker transport such as RabbitMQ or SQL Server, it is enough to specify the logical routing:
```

Comment:
> When running on a broker transport such as RabbitMQ or SQL Server

We are often providing the list of the broker based transports, often omitting one or another. Just an idea, what if we have a document (or an _include_ file/section) where we list all the supported transports, categorizing which ones are broker based, and only reference that rather than listing each tile all of them or a subset?